upgrading/topics/keycloak/changes.adoc

```diff
@@ -1,5 +1,16 @@
 == Migration Changes
 
+=== Migrating to 16.0.0
+
+==== Proxy environment variables
+
+{project_name} now respects the standard `HTTP_PROXY`, `HTTPS_PROXY` and `NO_PROXY` environment variables for outgoing
+HTTP requests. This change could lead to unexpected use of proxy server if you have for example `HTTP_PROXY` variable defined but have no
```

Comment:
use of "a" proxy server if you have for example "the" `HTTP_PROXY` variable

Reply:
Done.